_data/simple-icons.json

```diff
@@ -4112,6 +4112,11 @@
             "hex": "16A5F3",
             "source": "https://geticonjar.com/"
         },
+        {
+            "title": "Icons8",
+            "hex": "1FB041",
```

Comment:
From the website stylesheet, I found `#1FB141`. What do you think?

Reply:
Yes. I view code and see it is #1FB141. It's weird when I pick it with [Roy](https://www.useroy.com/), it got #1FB041. I think we should use #1FB141 as source code. I will update it.